api_feature.go

```diff
@@ -91,11 +91,11 @@ func (f *APIFeature) IPostToWithBody(path string, body *godog.DocString) error {
 	return f.makeRequest("POST", path, []byte(body.Content))
 }
 
-func (f *APIFeature) iPUT(path string, body *godog.DocString) error {
+func (f *APIFeature) IPut(path string, body *godog.DocString) error {
```

Comment:
As these are exported now, they'll need some GoDoc like the `IGet` and `IPost…` ones above.
```suggestion
// IPut makes a put request … etc. …
func (f *APIFeature) IPut(path string, body *godog.DocString) error {
```

Reply:
good point - GoDoc added now